### PR TITLE
Adaptions for reprocessing

### PR DESCRIPTION
**Pull request by skymandr**
This supersedes the previous #199 since that had an annoyingly messy commit-history.

@joakimmoller had  a comment about #199 that I still want to consider, but feel free to review in the meantime.

### CODE REVIEW

**Review by joakimmoller — APPROVED** [review]

File: src/mats_l1_processing/instrument.py [hunk 1]
```diff
@@ -332,29 +351,13 @@ def __init__(self, channel, calibration_file, start_datetime: datetime = datetim
...
-            WHERE datetime BETWEEN '{start_datetime}' AND '{end_datetime}'
-            AND channel = '{self.channel}'
-        '''
-        self.hot_pixels = pd.read_sql_query(query, disk_conn)
-        self.hot_pixels['datetime'] = pd.to_datetime(self.hot_pixels['datetime'])
+        self.hot_pixels = get_sqlite_data(filename, "hotpixelmaps")
         # Convert the HPM to a NumPy array
-        self.hot_pixels['HPM'] = self.hot_pixels['HPM'].apply(deserialize_blob)
-        disk_conn.close()
+        self.hot_pixels['HPM'] = self.hot_pixels['HPM'].apply(pickle.loads)
```
Comment (OleMartinChristensen) [inline src/mats_l1_processing/instrument.py:360]: pickle.loads did not work for me (when I tried), but perhaps I did something wrong..

Comment (skymandr) [reply src/mats_l1_processing/instrument.py:360]: Good to know!

Comment (skymandr) [reply src/mats_l1_processing/instrument.py:360]: Seems to work in the production environment, but would probably be good to test it locally as well.

**Comment by lindamegner** [comment]
I take it this is the hot pixel update. Happy that joakim has reviewed it too since this code is I dont follow everything in the code. But go ahead! :)

**Review by lindamegner — APPROVED** [review]
I take it this is the hot pixel update. Happy that joakim has reviewed it too since this code is I dont follow everything in the code. But go ahead! :)

**Comment by lindamegner** [comment]
@skymandr  Should this not have been merged prior to reprocessing?

**Comment by skymandr** [comment]
> @skymandr Should this not have been merged prior to reprocessing?

I'll merge it when you've verified the data looks all right, and we're ready to reprocess everything – this is the code currently running, so nothing to worry about. Merging etc. is just book keeping at this point.

**Pull request merged**